urdf/CMakeLists.txt

```diff
@@ -12,11 +12,24 @@ find_package(TinyXML REQUIRED)
 
 catkin_package(
   LIBRARIES ${PROJECT_NAME}
-  INCLUDE_DIRS include ${TinyXML_INLCLUDE_DIRS}
+  INCLUDE_DIRS include ${TinyXML_INLCLUDE_DIRS} ${CATKIN_DEVEL_PREFIX}/include
```

Comment:
This line is necessary to export the include path in devel-space context. However, I indeed missed to create that folder beforehand. I guess, it is created with `configure_file()`, but I moved the whole block past `catkin_package()` because of the use of `CATKIN_PACKAGE_INCLUDE_DESTINATION`.